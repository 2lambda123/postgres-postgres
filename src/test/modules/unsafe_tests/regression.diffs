diff -U3 /Users/kenaniah/workspace/postgres/src/test/modules/unsafe_tests/expected/role_membership.out /Users/kenaniah/workspace/postgres/src/test/modules/unsafe_tests/results/role_membership.out
--- /Users/kenaniah/workspace/postgres/src/test/modules/unsafe_tests/expected/role_membership.out	2021-10-17 12:34:10.000000000 -0700
+++ /Users/kenaniah/workspace/postgres/src/test/modules/unsafe_tests/results/role_membership.out	2021-10-17 12:46:31.000000000 -0700
@@ -189,11 +189,8 @@
 SET SESSION AUTHORIZATION role_g;
 GRANT pg_read_all_data TO role_f; -- error (no cluster-wide admin option)
 ERROR:  must have admin option on role "pg_read_all_data"
-REVOKE pg_read_all_data FROM role_f IN CURRENT DATABASE: -- success (database-specific admin option was inherited from role_b)
+REVOKE pg_read_all_data FROM role_f IN CURRENT DATABASE; -- success (database-specific admin option was inherited from role_b)
 GRANT pg_read_all_data TO role_f IN CURRENT DATABASE; -- success (database-specific admin option was inherited from role_b)
-ERROR:  syntax error at or near ":"
-LINE 1: REVOKE pg_read_all_data FROM role_f IN CURRENT DATABASE: 
-                                                               ^
 -- test REVOKE works
 -- test grant error (pre-existing)
 -- test revoke error (non-existing)
@@ -217,7 +214,7 @@
  pg_read_all_data | role_c | role_a     | f            | role_membership_test4
  pg_read_all_data | role_d | role_admin | t            | role_membership_test4
  pg_read_all_data | role_f | role_e     | f            | role_membership_test2
- pg_read_all_data | role_f | role_b     | f            | role_membership_test4
+ pg_read_all_data | role_f | role_g     | f            | role_membership_test4
  pg_read_all_data | role_f | role_e     | f            | 
  role_a           | role_d | role_admin | f            | role_membership_test1
  role_a           | role_e | role_admin | f            | role_membership_test1
