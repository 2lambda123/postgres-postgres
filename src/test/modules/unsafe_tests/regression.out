test rolenames                    ... ok          175 ms
test alter_system_table           ... ok           66 ms
test role_membership              ... FAILED      446 ms
